REST/PowerShell/Variables/GetVariableInProject.ps1

```diff
@@ -0,0 +1,106 @@
+Function Get-OctopusProject
```

Comment:
General formatting note: we can remove any duplicate whitespace/linebreaks.  The indention is wonky in Get-Variable too. Which editor are you using? If it's Code, I've got some suggestions for settings/extensions to help auto-format.